Ladybird/Tab.h

```diff
@@ -57,6 +58,7 @@ class Tab final : public QWidget {
 public slots:
     void focus_location_editor();
     void location_edit_return_pressed();
+    void open_file();
```

Comment:
Since we are calling Tab::open_file() from the BrowserWindow directly, I don't think it needs to be a slot and can just be a public method. The reason it needs to be a slot on BrowserWindow is that you're connecting it to the `QAction::triggered` signal from the QAction.